weaviate/connect/helpers.py

```diff
@@ -35,12 +39,32 @@ def connect_to_wcs(
     Returns
         `weaviate.WeaviateClient`
             The client connected to the cluster with the required parameters set appropriately.
+
+    Examples:
```

Comment:
I feel like this is maybe the only place where it would be useful to have it in both locations since it's the first thing users do and will want as much guidance as possible. Especially given the need to call `.connect()` or use the context manager